src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
Add variable for your color `#00acdc;` here because the color often repeat in the code. And use variable when you need